test/passing/infix_bind.ml

```diff
@@ -181,3 +181,28 @@ let foo =
       let batch = {repo; tree= old_tree; origin} in
       let b = Batch batch in
       foo
+
+let _ = foo >>= function[@warning "-4"] A -> false | B -> true
+
+let _ =
+  foo >>= function[@warning "-4"]
+  | Afoooooooooooooooooo fooooooooo -> false
+  | Bfoooooooooooooooooooooo fooooooooo -> true
+
+let _ =
+  foo >>= fun [@warning "-4"] x ->
+  fooooooooooooooooooooooo
+
+let _ =
+  foo >>= fun [@warning "-4"] x y ->
+  fooooooooooooooooooooooo fooooooooooooooooooooooo fooooooooooooooooooooooo
+    fooooooooooooooooooooooo
+
+let _ =
+  foo >>= function(* foo before *) [@warning "-4"] (* foo after *)
```

Comment:
Shouldn't be a space after `function` ?

Reply:
It wouldn't hurt to have a space after the keyword, but there is also no space when `break-infix-before-func=true` so I was thinking at looking at that in another PR (didn't find a quick way to find that just now).